README.md

```diff
@@ -71,6 +71,11 @@ A curated list of insanely awesome libraries, packages and resources for Quants
 - [moonshot](https://github.com/quantrocket-llc/moonshot) - Vectorized backtester and trading engine for QuantRocket based on Pandas.
 - [PyPortfolioOpt](https://github.com/robertmartin8/PyPortfolioOpt) - Financial portfolio optimisation in python, including classical efficient frontier and advanced methods.
 - [mlfinlab](https://github.com/hudson-and-thames/mlfinlab) - Implementations regarding "Advances in Financial Machine Learning" by Marcos Lopez de Prado. (Feature Engineering, Financial Data Structures, Meta-Labeling)
+- [aat](https://github.com/timkpaine/aat) - Async Algorithmic Trading Engine
+- [backtesting.py](https://github.com/kernc/backtesting.py) - Backtest trading strategies in Python
```

Comment:
Thanks! Personally, I'd prefer for _Backtesting.py_ to be pointed to its [canonical homepage](https://kernc.github.io/backtesting.py/). Also, _Backtesting.py_ (capitalized) in prose. Description could be extended / improved upon. 😁

Reply:
URL and first letter fixed @kernc